bundle/manifests/sailoperator.clusterserviceversion.yaml

```diff
@@ -285,6 +289,10 @@ spec:
         - urn:alm:descriptor:com.tectonic.ui:select:v1.22.5
         - urn:alm:descriptor:com.tectonic.ui:select:v1.21.6
         - urn:alm:descriptor:com.tectonic.ui:select:v1.25-alpha.df953ef6
+        - urn:alm:descriptor:com.tectonic.ui:select:master
+        - urn:alm:descriptor:com.tectonic.ui:select:v1.24-latest
+        - urn:alm:descriptor:com.tectonic.ui:select:v1.23-latest
+        - urn:alm:descriptor:com.tectonic.ui:select:v1.22-latest
```

Comment:
This is what I was talking about. The version dropdown list now has this weird order:
- v1.24.2
- v1.24.1
- v1.24.0
- v1.23.4
- ...
- v1.23.0
- v1.24-latest
- v1.23-latest

We probably want the order to be the following:
- v1.24-latest
- v1.24.2
- v1.24.1
- v1.24.0
- v1.23-latest
- v1.23.4
- v1.23.3
- ...

Reply:
Probably not, since the numbers don't have leading zeroes and lexicographical sorting would produce this:
- v1.24.9
- v1.24.8
- v1.24.11
- v1.24.10

Reply:
I have updated the `versions.yaml` and it should display it as the wanted order now.